.ci/pipeline/release_matrix_job.yaml

```diff
@@ -32,8 +32,16 @@ runs_on_dockers:
       tag: '20241028'
     }
 
-
 steps:
+  - name: Remove old weekly folder
```

Comment:
this logic should be in the launcher, its a very specific step for weekly process, so launcher should do all preparations.
I want to avoid changes to release job if not necessary.

Reply:
Removed cleanups